packages/gafl-webapp-service/src/pages/order-complete/order-complete/__tests__/order-complete.spec.js

```diff
@@ -1,42 +1,56 @@
-import { salesApi } from '@defra-fish/connectors-lib'
-import { displayStartTime } from '../../../../processors/date-and-time-display.js'
-import { initialize, injectWithCookies, start, stop, mockSalesApi } from '../../../../__mocks__/test-utils-system'
-import { JUNIOR_LICENCE } from '../../../../__mocks__/mock-journeys.js'
-import {
-  AGREED,
-  ORDER_COMPLETE,
-  TERMS_AND_CONDITIONS,
-  LICENCE_DETAILS,
-  COOKIES,
-  ACCESSIBILITY_STATEMENT,
-  PRIVACY_POLICY,
-  REFUND_POLICY,
-  NEW_TRANSACTION
-} from '../../../../uri.js'
+import { ORDER_COMPLETE, LICENCE_DETAILS, NEW_TRANSACTION } from '../../../../uri.js'
 import { addLanguageCodeToUri } from '../../../../processors/uri-helper.js'
```

Comment:
are we now not using addLanguageCodeToUri as per: https://github.com/DEFRA/rod-licensing/pull/1580

Reply:
It's still needed here, I think, because we're putting links in the html rather than doing a redirect.